### PR TITLE
Merging Intake into Master

### PR DESCRIPTION
**Pull request by daija100**

### CODE REVIEW

File: BragiCode/src/main/java/frc/robot/subsystems/Intake.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Intake extends SubsystemBase {
+  public TalonFX IntakeM1;
+  public TalonFX IntakeM2;
+  public TalonFX IntakeM3;
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Intake.java:17]: The motors should have more descriptive names. We're going to want to know which motor is in which position.

File: BragiCode/src/main/java/frc/robot/subsystems/Intake.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  public enum IntakeState {
+    S_WaitingForBall, S_PopOutIntake, S_WaitingForShooter
+  }
+
+  public Intake() {
+    IntakeM1 = new TalonFX(Constants.IntakeConstants.IntakeMotor1);
+    IntakeM2 = new TalonFX(Constants.IntakeConstants.IntakeMotor2);
+    IntakeM3 = new TalonFX(Constants.IntakeConstants.IntakeMotor3);
+
+  var slot0Configs = new Slot0Configs();
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Intake.java:33]: indent
```suggestion
    var slot0Configs = new Slot0Configs();
```

File: BragiCode/src/main/java/frc/robot/subsystems/Intake.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      mIntakeState = IntakeState.S_WaitingForShooter;
+    }
+  }
+
+  public void WaitingForShooter() {
+    IntakeM1.set(0);
+    IntakeM2.set(0);
+  }
+
+  public double IntakePos(TalonFX motorFx)
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Intake.java:72]: IntakePos should be `private`

```suggestion
  private double IntakePos(TalonFX motorFx)
```

**Comment by duane701** [comment]
FTP

A few comments to address.

**Comment by daija100** [comment]
PTAL

File: BragiCode/src/main/java/frc/robot/subsystems/Intake.java [hunk 4]
```diff
@@ -52,32 +51,33 @@ public void RunIntakeState() {
...
   }
-  public void Position(){
-    SmartDashboard.putNumber("IntakeMotor1Position",IntakePos(IntakeM1));
-    SmartDashboard.putNumber("IntakeMotor2Position",IntakePos(IntakeM2));
-    SmartDashboard.putNumber("IntakeMotor3Position",IntakePos(IntakeM3));
+
+  public void Position() {
+    SmartDashboard.putNumber("IntakeMotor1Position", IntakePos(IntakeMotorRight));
+    SmartDashboard.putNumber("IntakeMotor2Position", IntakePos(IntakeMotorLeft));
+    SmartDashboard.putNumber("IntakeMotor3Position", IntakePos(IntakePopMotor));
```
Comment (duane701) [inline BragiCode/src/main/java/frc/robot/subsystems/Intake.java:78]: Give the dashboard descriptive names.

The dashboard text is a label. We can use Title Case. For example `Intake Motor Pop`. Looks spaces and everything!

File: BragiCode/src/main/java/frc/robot/subsystems/Intake.java (hunk 1 above)
Comment (duane701) [reply BragiCode/src/main/java/frc/robot/subsystems/Intake.java]: Renamed in several places.

**Comment by duane701** [comment]
FTP

**Comment by daija100** [comment]
PTAL

**Comment by duane701** [comment]
LGTM

**Pull request merged**